README.md

```diff
@@ -1 +1,58 @@
-# JS-Capstone-Project-
\ No newline at end of file
+# Javascript Capstone Project
+> An API based web app that displays Meals from the MealsDB API and the Involvement API from Microverse. The features of this webapp include
+
+An option to like a meal by clicking on the like button
+A comment button that opens up a modal and allows users to leave comments.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript(ES6)
+- Webpack
+- Jest Library
+
+## Live Demo
+- at http://localhost:8080/
+
+## Getting Started
+
+### To get a local copy run the following steps:
+
+- [Copy this link](https://github.com/mbdesigns1989/To-do-List/tree/To-Do-list-add-and-remove)
+- Open your terminal or command line
+- Run `git clone` and paste the link
+- Open the folder with your code editor
+- Create a branch to work on
+- Run `npm install`
+- Run `npm run build`
+- Run `npm start` and it will load the app in your default browser.
+- Run `npm run start` to launch the server.
+- Run `npm run test` to run jest test cases.
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: (https://github.com/mbdesigns1989)
+- Twitter: (https://twitter.com/mohamed66083152)
+- LinkedIn: (https://www.linkedin.com/in/mohamed-badr-27b26a212/)
+
+## Author 2
+
+GitHub: (https://github.com/Busiwa24) 
+Twitter: (https://twitter.com/busiwaliuma) 
+LinkedIn: (https://www.linkedin.com/in/busiwa-liuma-28385b80/>)
+
```

Comment:
- [ ] Add `-` before the social media accs to look the same as we have in the first author.

Reply:
Well solved